test/integration/workarea/storefront/sharing_integration_test.rb

```diff
@@ -32,7 +32,7 @@ def test_sends_a_sharing_email
         end
 
         assert_includes(email.to, to_email)
-        assert_includes(email.from, Workarea.config.email_from)
+        assert_match(email.from.first, Workarea.config.email_from)
```

Comment:
We are mentioning the from_email = 'bcrouse@workarea.com'
Does Workarea.config.email_from override the above ?

Reply:
The `reply_to` is set to that `from_email` above, we could only send the email from the email address the system is allowed to send from.

Reply:
@meowsus is `email.from` an array? if so, matching on the first value only actually makes it less permissive... and why `assert_match` instead of `assert_equal`?

Maybe something like `assert(email.from.include? { |e| e =~ Workarea.config.email_from })` if it is an array.

Reply:
Yes, `email.from` is an array. What I was seeing is `Workarea.config.email_from` coming back as `Some Name <some.name@domain.com>`.